test/scripts/shifu-demo-install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+set -e
+set -o noglob
+
+# get info of System
+setup_verify_os_and_arch() {
+    OS=$(uname -s | tr "[:upper:]" "[:lower:]" ) 
+    ARCH=$(uname -m)
+    case $ARCH in
+        amd64)
+            ARCH=amd64
+            ;;
+        x86_64)
+            ARCH=amd64
+            ;;
+        arm64)
+            ARCH=arm64
+            ;;
+        aarch64)
+            ARCH=arm64
+            ;;
+        arm*)
+            ARCH=arm64
+            ;;
+        *)
+            fatal "Unsupported architecture $ARCH"
+    esac
+}
+
+setup_verify_os_and_arch
+
+curl -LO https://demo.shifu.run/demo-content/shifu_demo_aio_${OS}_${ARCH}.tar
+
+rm -rf testdir && mkdir testdir
+
+tar -xvf shifu_demo_aio_${OS}_${ARCH}.tar -C testdir 
+
+cd testdir
+
+bash scripts/deviceshifu-demo-aio.sh run_demo
```

Comment:
need to update this to `chmod +x scripts/deviceshifu-demo-aio.sh && sudo ./scripts/deviceshifu-demo-aio.sh run_demo`